environment.yml

```diff
@@ -0,0 +1,105 @@
+  - certifi=2020.12.5=py39h6e9494a_1
+  - cfitsio=3.470=h01dc385_7
+  - cftime=1.3.1=py39hae3b40e_0
+  - curl=7.71.1=hcb81553_8
+  - dcw-gmt=1.1.4=1001
+  - expat=2.2.9=hb1e8313_2
+  - ffmpeg=4.3.1=hb787a4c_2
+  - fftw=3.3.9=nompi_h723ed24_100
+  - fontconfig=2.13.1=hd23ceaa_1004
+  - freetype=2.10.4=h4cff582_1
+  - freexl=1.0.5=h0b31af3_1002
+  - gdal=3.1.4=py39h906d53f_0
+  - geos=3.8.1=h4a8c4bd_0
+  - geotiff=1.6.0=hc1004f4_3
+  - gettext=0.19.8.1=h7937167_1005
+  - ghostscript=9.53.3=h2e338ed_2
+  - giflib=5.2.1=hbcb3906_2
+  - glib=2.66.4=h22858aa_1
+  - gmp=6.2.1=h2e338ed_0
+  - gmt=6.1.1=haa38f90_2
+  - gnutls=3.6.13=h756fd2b_1
+  - gshhg-gmt=2.3.7=1002
+  - hdf4=4.2.13=h71d84a9_1004
+  - hdf5=1.10.6=nompi_hc5d9132_1114
+  - icu=67.1=hb1e8313_0
+  - jpeg=9d=hbcb3906_0
+  - json-c=0.13.1=h575e443_1002
+  - kealib=1.4.14=h82515a5_0
+  - krb5=1.17.2=h60d9502_0
+  - lame=3.100=h35c211d_1001
+  - libblas=3.9.0=7_openblas
+  - libcblas=3.9.0=7_openblas
+  - libcurl=7.71.1=h9bf37e3_8
+  - libcxx=11.0.0=h4c3b8ed_1
+  - libdap4=3.20.6=h993cace_1
+  - libedit=3.1.20191231=h0678c8f_2
+  - libev=4.33=haf1e3a3_1
+  - libffi=3.3=h046ec9c_2
+  - libgdal=3.1.4=hd7bf8dc_0
+  - libgfortran=5.0.0=9_3_0_h6c81a4c_16
+  - libgfortran5=9.3.0=h6c81a4c_16
+  - libglib=2.66.4=h7424822_1
+  - libiconv=1.16=haf1e3a3_0
+  - libkml=1.3.0=h8ca2c65_1012
+  - liblapack=3.9.0=7_openblas
+  - libnetcdf=4.7.4=nompi_h9d8a93f_107
+  - libnghttp2=1.41.0=h7580e61_2
+  - libopenblas=0.3.12=openmp_h54245bb_1
+  - libpng=1.6.37=h7cec526_2
+  - libpq=12.3=h7049927_3
+  - libspatialite=5.0.0=hc85de81_0
+  - libssh2=1.9.0=h8a08a2b_5
+  - libtiff=4.2.0=h355d032_0
+  - libwebp-base=1.1.0=hbcb3906_3
+  - libxml2=2.9.10=h2c6e4a5_2
+  - llvm-openmp=11.0.1=h7c73e74_0
+  - lz4-c=1.9.3=h046ec9c_0
+  - ncurses=6.2=h2e338ed_4
+  - netcdf4=1.5.5.1=nompi_py39he9bd12f_100
+  - nettle=3.6=hedd7734_0
+  - numpy=1.19.5=py39he588a01_1
+  - openh264=2.1.1=hfd3ada9_0
+  - openjpeg=2.3.1=h6cbf5cd_3
+  - openssl=1.1.1i=h35c211d_0
+  - packaging=20.8=pyhd3deb0d_0
+  - pandas=1.2.0=py39h089d6f7_1
+  - pcre=8.44=hb1e8313_0
+  - pip=20.3.3=pyhd8ed1ab_0
+  - pixman=0.40.0=hbcb3906_0
+  - poppler=0.89.0=h9de36d4_1
+  - poppler-data=0.4.10=0
+  - postgresql=12.3=he852019_3
+  - proj=7.1.1=h45baca5_3
+  - pygmt=0.2.1=pyhd8ed1ab_1
+  - pyparsing=2.4.7=pyh9f0ad1d_0
+  - python=3.9.1=h1d169a7_3_cpython
+  - python-dateutil=2.8.1=py_0
+  - python_abi=3.9=1_cp39
+  - pytz=2020.5=pyhd8ed1ab_0
+  - readline=8.0=h0678c8f_2
+  - setuptools=49.6.0=py39h6e9494a_3
+  - six=1.15.0=pyh9f0ad1d_0
+  - sqlite=3.34.0=h17101e1_0
+  - tiledb=2.1.6=h6a9efa5_1
+  - tk=8.6.10=h0419947_1
+  - tzcode=2020f=h35c211d_0
+  - tzdata=2020f=he74cb21_0
+  - wheel=0.36.2=pyhd3deb0d_0
+  - x264=1!161.3030=h35c211d_0
+  - xarray=0.16.2=pyhd8ed1ab_0
+  - xerces-c=3.2.3=h0a0444a_1
+  - xz=5.2.5=haf1e3a3_1
+  - zlib=1.2.11=h7795811_1010
+  - zstd=1.4.8=hf387650_1
+prefix: /Users/jlmd9g/software/miniconda3/envs/2D_strain
```

Comment:
is this script able to 'pip install Tectonic-Utils' as well? It would be awesome to do that if possible